### ISSUE TITLE
training metrics

### ISSUE THREAD

**Issue by sonal-bansal** [andy-yun/pytorch-0.4-yolov3#90]
The console prints a few metrics while training like nGT, nRC, nRC75, nPP, box loss, conf loss, class loss, total loss, I have following assumptions : 
1) nGT : it counts the number of ground truths (labels) provided by us for each pic, nGT is simply a count of those.
2) nRC : is the classic recall = true positive / (true condition positive) which is > 0.5, here, I dont know how what does 'n' means in this context.
3) nRC75 : is same as nRC but only counted for > 0.75
4) nPP : it is the number of regions proposed, but is it safe to assume that these are the bounding boxes proposed by the model as the output?
5) class loss : it is the BCE with logits loss for each class, so no confusion here.
6) total loss : it is the sum of all the losses mentioned(is it right?)

I have no clue about box loss and conf loss, secondly are my assumptions above correct?

Please help, thanks in advance.

**Comment by andy-yun** [comment]
@sonal-bansal You're right. See the equation (3) of https://arxiv.org/pdf/1506.02640.pdf .
box loss means the sum of coordination and box size differences.
conf loss means the prediction loss.
